### PR TITLE
Fix loading of PasswordExecCommand in UI while importing server and running PasswordExecCommand in server mode. #6792

### PR DESCRIPTION
**Pull request by anilsahoo20**

### CODE REVIEW

File: web/pgadmin/utils/__init__.py [hunk 1]
```diff
@@ -491,7 +491,8 @@ def dump_database_servers(output_file, selected_servers,
             add_value(attr_dict, "ConnectionParameters",
                       server.connection_params)
 
-            # if desktop mode
+            # if desktop mode or server mode with
+            # config['ENABLE_SERVER_PASS_EXEC_CMD'] = False
```
Comment (adityatoshniwal) [inline web/pgadmin/utils/__init__.py:495]: remove commented code.

File: web/pgadmin/utils/__init__.py [hunk 2]
```diff
@@ -727,8 +728,10 @@ def load_database_servers(input_file, selected_servers,
 
             new_server.kerberos_conn = obj.get("KerberosAuthentication", None)
 
-            # if desktop mode
-            if not current_app.config['SERVER_MODE']:
+            # if desktop mode or server mode with
+            # config['ENABLE_SERVER_PASS_EXEC_CMD'] = False
```
Comment (adityatoshniwal) [inline web/pgadmin/utils/__init__.py:732]: remove commented code.

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]

File: web/pgadmin/utils/__init__.py (hunk 2 above)
Comment (anilsahoo20) [reply web/pgadmin/utils/__init__.py]: Corrected

File: web/pgadmin/utils/__init__.py (hunk 1 above)
Comment (anilsahoo20) [reply web/pgadmin/utils/__init__.py]: Corrected

**Pull request merged**